src/test/compilation-failed/contracts/const-eval-constant-circular-dependency-with-recursive-function.tact

```diff
@@ -0,0 +1,15 @@
+const A: Int = foo(20);
```

Comment:
let's have both an error message for `foo(3)` and `foo(20)`

Reply:
Done